src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/AbstractFilePathService.cs

```diff
@@ -45,6 +45,12 @@ private string GetRazorFilePath(string filePath)
         {
             trimIndex = filePath.LastIndexOf(_languageServerFeatureOptions.CSharpVirtualDocumentSuffix);
 
+            if (trimIndex == -1)
+            {
+                // Not a C# file, and we wouldn't have got here if it wasn't Html, so nothing left to do
```

Comment:
nit: Wouldn't have gotten here if it "was" html right?

Reply:
Whoops, yes
```suggestion
                // Not a C# file, and we wouldn't have got here if it was Html, so nothing left to do
```